addons/medical_treatment/ACE_Medical_Treatment_Actions.hpp

```diff
@@ -14,6 +14,16 @@ class ACEGVAR(medical_treatment,actions) {
         treatmentTime = 5;
     };
 
+    class Morphine;
+    class Apap: Morphine {
+        allowedSelections[] = {"head"};
+        displayName = CSTRING(Inject_Apap);
+        displayNameProgress = CSTRING(Injecting_Apap);
+        items[] = {"ACE_apap"};
```

Comment:
Why are we using ACE tag here?
```suggestion
        items[] = {"AFMED_apap"};
```

Reply:
I didn't know if we should change this from legacy version. Will fix this.

Reply:
It will break old missions but whatever... ACEs tag should not have been used in first place.